src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -329,11 +329,12 @@ internal class RecommendationService(
     featureFlags: FeatureFlags?,
   ): RecommendationResponse {
     validateRecallType(jsonRequest)
+    val requestJson = JSONObject(jsonRequest.toString())
     val existingRecommendationEntity = getRecommendationEntityById(recommendationId)
     val updatedRecommendation: RecommendationModel =
       recommendationFromRequest(existingRecommendationEntity, jsonRequest)
+    updatedRecommendation.isUnder18 = updatedRecommendation.isOver18 // uses isOver18 field which is inverted and still used by front end
```

Comment:
This looks wrong. 

Today, the FE asks - is the offender under 18? - it then saves this as a boolean value in the rec doc. as in...
recommendation.isOver18 = ! postData.isUnder18

consequently  if the offender is 18, then isUnder18 = false, and therefore isOver18 = true.

Reply:
Are you sure you have the line right?

Reply:
corrected